examples/systemd/99-udc.rules

```diff
@@ -0,0 +1,5 @@
+#  SPDX-License-Identifier: LGPL-2.1+
```

Comment:
Really LGPL license for examples?

Reply:
Actually not for all the examples (the .rst is CC-BY-SA), but for systemd-related files. LGPL-2.1+ is what systemd uses so it is a matter of simply applying the same style. What would otherwise be in line with gt's spirit?

Reply:
Re-licensed.